src/routes/Dashboard/components/ForecastTile/ForecastTile.jsx

```diff
@@ -39,7 +40,14 @@ const ForecastTile = ({ weather }) => {
         <TableRow key={i}>
           <TableRowColumn style={styles.column}>{forecast.dow}</TableRowColumn>
           <TableRowColumn style={styles.column}>
+            <img
```

Comment:
nice that we have icons here now. Can you add it to the ShipmentCard too for consistency?